local-config/docker-compose.testrunner.yaml

```diff
@@ -18,3 +18,21 @@ services:
         condition: service_healthy
       demo-rpc:
         condition: service_healthy
+      - tracetest
```

Comment:
this line looks like an merge error, it's already defined above to use the health check

Reply:
you are right. done